eth_tester/backends/mock/factory.py

```diff
@@ -259,6 +259,15 @@ def make_genesis_block():
         "uncles": [],
     }
 
+    if overrides is not None:
+        if not (all(bool(override in genesis_block) for override in overrides)):
+            fields = ', '.join(genesis_block)
+            error = "Invalid genesis overrides; Valid parameters are {}".format(fields)
+            raise ValueError(error)
+        genesis_block.update(overrides)
```

Comment:
Very minor but we prefer non-mutative approaches.  You can use `cytoolz.merge` as follows.

```python
from cytoolz import merge
genesis_block = merge(default_genesis_block, overrides)
```